R/homoSapiens/humanAtlas.r

```diff
@@ -58,7 +58,33 @@ appendGenes <- function(humanGenesVector, GSEMatrix)
   return(fullMatrix) #Return fullMatrix
 }
 
+checkRawCounts <- function(GSEMatrix)
+{
+  GSEBigMatrix <- as.big.matrix(GSEMatrix)
```

Comment:
I'm not sure what `as.big.matrix` does here.

Reply:
I think I read that the is.float() function is part of the bigmemory library and only works on type big.matrix so I had to convert GSEMatrix to a big.matrix. Probably is not necessary now due to the float library.

Reply:
see comment below, I think you'd be better off using base R functions for this.

Reply:
Yea looks like it is testing for 32bit precision values, which by default a numeric matrix I think will have 64bit, so it will not detect most numeric (double precision) values. 

``` r
library(float)
x = matrix(1.23, 5, 5)
x
#>      [,1] [,2] [,3] [,4] [,5]
#> [1,] 1.23 1.23 1.23 1.23 1.23
#> [2,] 1.23 1.23 1.23 1.23 1.23
#> [3,] 1.23 1.23 1.23 1.23 1.23
#> [4,] 1.23 1.23 1.23 1.23 1.23
#> [5,] 1.23 1.23 1.23 1.23 1.23
is.float(x)
#> [1] FALSE
```

<sup>Created on 2020-07-07 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Reply:
Ok then let's use default R functions for this task. `is.Integer` and `is.double` might be useful here since we wouldn't have to worry about the function detecting 64 bit floats.

Reply:
See commit https://github.com/rnabioco/scRNA-seq-Cell-Reference-Matrix/pull/19/commits/020060bd3937512cf1d9dffa4697bf06cc15bb2e